documentation/Demo-Server-Guide.md

```diff
@@ -269,11 +268,27 @@ From there the following commands might be useful:
 
 1. Log in to the server using terminal
 2. Copy the log to an accessible place (in the server) i.e. if getting log for instance 5000:
-`sudo cp /var/lib/docker/volumes/conforma-on-8000_logs/_data/server.log demo_server/2022-11-15.log`
-3. Now exit the server `exit` and on your local machine open the terminal 
+   `sudo cp /var/lib/docker/volumes/conforma-on-8000_logs/_data/server.log demo_server/2022-11-15.log`
```

Comment:
Should be a line break here (two spaces at the end, and match the indentation -- check it in Preview)